otherlibs/stdune/src/compact_position.mli

```diff
@@ -0,0 +1,47 @@
+(** Positions information that can be encoded within a single immediate *)
+
+type t [@@immediate]
+
+val of_position : Lexbuf.Position.t -> t
```

Comment:
This could perhaps be `of_lexbuf_position` to avoid confusion if you agree to rename this module to just `Position`.